plugins/modules/find.py

```diff
@@ -16,11 +16,16 @@
 
 # If this is part of a collection, you need to use semantic versioning,
 # i.e. the version is of the form "2.5.0" and not "2.4".
-version_added: "0.1.0"
+version_added: "0.1.1"
```

Comment:
This field defines the version of the collection in which this module was introduced. It should not be updated.
```suggestion
version_added: "0.1.0"
```